sidebars.js

```diff
@@ -851,6 +852,841 @@ const sidebars = {
+                  label: 'Helm Chart を使用した TLS 対応 ScalarDB Cluster',
+                },
+                {
+                  type: 'doc',
+                  id: 'helm-charts/getting-started-scalardb-cluster-tls-cert-manager',
+                  label: 'cert-manager と Helm Chart を使用した TLS 対応 ScalarDB Cluster',
+                },
+                {
+                  type: 'doc',
+                  id: 'helm-charts/getting-started-scalardb-analytics-postgresql',
+                  label: 'Helm Chart を使用した ScalarDB Analytics with PostgreSQL',
+                },
+                {
+                  type: 'doc',
+                  id: 'helm-charts/getting-started-monitoring',
+                  label: 'Helm Chart を使用した監視用 Prometheus Operator',
+                },
+                {
+                  type: 'doc',
+                  id: 'helm-charts/getting-started-logging',
+                  label: 'Helm Chart を使用したログ記録用 Loki スタック',
+                },
+                {
+                  type: 'doc',
+                  id: 'helm-charts/getting-started-scalar-manager',
+                  label: 'Helm Chart を使用した Scalar Manager',
+                },
+              ],
+            },
+            {
+              type: 'category',
+              label: 'データベースセットアップガイド',
+              collapsible: true,
+              items: [
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/SetupDatabaseForAWS',
+                  label: 'AWS',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/SetupDatabaseForAzure',
+                  label: 'Azure',
+                },
+              ],
+            },
+            {
+              type: 'category',
+              label: 'インストールガイド',
+              collapsible: true,
+              items: [
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/HowToGetContainerImages',
+                  label: 'コンテナイメージを取得する',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/AwsMarketplaceGuide',
+                  label: 'AWS Marketplace からインストール',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/HowToUseContainerImages',
+                  label: 'コンテナイメージを使用',
+                },
+              ],
+            },
+            {
+              type: 'category',
+              label: '展開ガイド',
+              collapsible: true,
+              items: [
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/CreateEKSClusterForScalarDBCluster',
+                  label: 'ScalarDB Cluster 用の EKS クラスターを作成する',
+                },
+                {
+                  type: 'doc',
+                  id: 'scalar-kubernetes/CreateBastionServer',
+                  label: '要塞サーバーを作成する',
```

Comment:
```suggestion
                  label: '踏み台サーバーを作成する',
```

I think `踏み台サーバー` is a common term for bastion servers.